gossip/src/ping_pong.rs

```diff
@@ -25,7 +25,7 @@ pub struct Ping<T> {
     signature: Signature,
 }
 
-#[derive(AbiExample, Debug, Deserialize, Serialize)]
+#[derive(AbiExample, Debug, Default, Deserialize, Serialize)]
```

Comment:
Again `Default` here is meaningless.
Can we not add it to the API?

Reply:
are you looking for a `_for_tests()` API?